release-notes/WCF-Web-Service-Reference-notes.md

```diff
@@ -4,6 +4,9 @@ WCF Web Service Reference connected services provider is now part of Visual Stud
 
 ## Release History
 
+### 4/4/2018 - Visual Studio 2017 v15.7 Preview 3.0 [27602.0.d15.7]
+* Implemented full support for future framework versions ([#2719](https://github.com/dotnet/wcf/issues/2719))
```

Comment:
Feels a bit confusing when comparing this with the release notes of preview 2.0. This sounds like just bug fix.